.npmrc

```diff
@@ -0,0 +1,9 @@
+# rollup-plugin-dts needs packages to be hoisted
```

Comment:
It's the only part with "not obvious" settings:
- it may deserve more visibility (I won't look there at first, when having a build error),
maybe a "troubleshouting" section in readme.
- is it really mandatory?
![image](https://user-images.githubusercontent.com/62407/158963777-12d96762-fc0c-4aca-9815-4fc8a3d73e57.png)

Reply:
same as https://github.com/scaleway/scaleway-ui/pull/1245#discussion_r829798407

Ok, I got an error, this is required to build ;)
![image](https://user-images.githubusercontent.com/62407/158967516-6c6feeca-0056-4785-a58a-71e0307774cf.png)
once I trashed node_modules and made a fresh reinstall with pnpm!

=> It may **deserve more visibility** (I won't look there at first, when having a build error),
**maybe a "troubleshouting" section in readme**.